src/main/scala/com/fulcrumgenomics/commons/util/SystemUtil.scala

```diff
@@ -58,7 +58,7 @@ object SystemUtil {
   /** True if the current system might support the Intel Inflater and Deflater, false otherwise. */
   lazy val IntelCompressionLibrarySupported: Boolean = {
     if (!isLinux && !isMac) false
-    else if (isOsArch(name = "ppc64le")) false
+    else if (isMac && !isOsArch("x86")) false
     else if (isMac && isOsVersion("10\\.14\\..+".r)) false // FIXME: https://github.com/Intel-HLS/GKL/issues/101
```

Comment:
I don't know if anyone is living in the past so it could be fine to leave this here. I'm up to `11.6.2` now.